ci-operator/step-registry/hypershift-qe/install/hypershift-qe-install-commands.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -euo pipefail
+
+BUCKETNAME="hypershiftqe$RANDOM"
+
+bucket_name_file="$SHARED_DIR/bucket_name"
+echo $BUCKETNAME > "$bucket_name_file"
+
+function export-credentials() {
```

Comment:
All the credenteials are already existing along with CLUSTER_PROFILE, https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L269-L287, do we need to extract them again?